### ISSUE TITLE
Bug Report (Submitted by 3723)

### ISSUE THREAD

**Issue by GuthrieW** [GuthrieW/shl-trading-cards#222]
## Description
Binder update page is not fully keyboard accessible. You cannot access the cards to add them just via keyboad. https://imgur.com/a/QqCvABa
    
## Reproduction Steps
Went to the "update binder" page on one of my existing binder and tabbed through the interface via keyboard. All the cards are skipped so they can only be clicked on with a mouse.
    
## Operating System
Windows
    
## Browser
Chrome
    
## Device
Desktop
